src/components/Pagination/Pagination.scss

```diff
@@ -96,7 +98,7 @@ $prefixClass: rc-pagination;
       display: block;
       letter-spacing: 2px;
       color: #ccc;
-      font-size: 12px;
+      font-size: $font-size;
```

Comment:
@Farhan-emgage I don't think this is necessary. It was already specified on the parent class (line 17)